File: Jimmy Selgen Nielsen/MessageAggregator/AirlineMessageSplitter - Java/src/main/java/com/example/airlinesplitter/messages/Luggage.java
package com.example.airlinesplitter.messages;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;
import jakarta.xml.bind.annotation.XmlElement;
import jakarta.xml.bind.annotation.XmlRootElement;
import lombok.Data;
import lombok.extern.jackson.Jacksonized;

import java.util.UUID;

@Data
@Jacksonized
@XmlRootElement(name = "Luggage")
@XmlAccessorType(XmlAccessType.FIELD)
public class Luggage {
    @XmlElement(name = "id")
    private String id;
    @XmlElement(name = "identification")
    private String identification;
    @XmlElement(name = "category")
    private String category;
    @XmlElement(name = "weight")
    private Double weight;
    private UUID passengerId;
    private UUID flightId;
}

File: Jimmy Selgen Nielsen/MessageAggregator/AirlineMessageSplitter - .NET/Messages/Luggage.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml.Serialization;

namespace AirlineMessageSplitter.Messages
{
	/*
	   <Luggage>
		<Id>CA937200305252</Id>
		<Identification>2</Identification>
		<Category>Large</Category>
		<Weight>22.7</Weight>
	   </Luggage>
	 */

	public class Luggage
	{
		
		[XmlElement(ElementName = "Id")]
		public string id;
		[XmlElement(ElementName = "Identification")]
		public string identification;
		[XmlElement(ElementName = "Category")]
		public string category;
		[XmlElement(ElementName = "Weight")]
		public double weight;
		[XmlIgnore]
		public Guid passengerId;
		[XmlIgnore]
		public Guid flightId;
	}
}
